include/sea_dsa/ShadowMem.hh

```diff
@@ -30,7 +49,24 @@ public:
 	    llvm::Pass &pass /* for dominatorTree and assumptionCache*/,
 	    bool splitDsaNodes = false, bool computeReadMod = false,
```

Comment:
is `computeReadMod` used? It should be taken care of by `sea-dsa`. This was a limitation of `llvm-dsa`

Reply:
If `sea-dsa` is context-insensitive then I think it still makes sense to use `computeReadMod`. But at least we should allow user to enable it only sea-dsa is context-insensitive

Reply:
Do you mean that the new flow-sensitivity of tea-dsa should subsume `computeReadMod`?